applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -320,12 +349,168 @@ def test_TL_3D_hexa(self):
         mp.CreateNewElement("TotalLagrangianElement3D8N", 6,[11,4,9,12,14,8,15,16], mp.GetProperties()[1])
         mp.CreateNewElement("TotalLagrangianElement3D8N", 7,[11,12,10,3,14,16,13,6], mp.GetProperties()[1])
 
-        A,b = self._define_movement(dim)
+        A,b = self._define_movement_TL(dim)
         
         self._apply_BCs(bcs,A,b)
         self._solve(mp)
         self._check_results(mp,A,b)
         self._check_outputs(mp,A,dim)
+        
+    #def test_UL_2D_triangle(self):
```

Comment:
This is right the problem with the test

the previous displacement and the current displacement is the same